config.js

```diff
@@ -29,6 +29,11 @@ const config = {
       webflow: {
         collection: env('FOXY_WEBFLOW_COLLECTION'),
         token: env('FOXY_WEBFLOW_TOKEN') || env('WEBFLOW_TOKEN'),
+      },
+      wix: {
+        accountId: env("FOXY_WIX_ACCOUNT_ID"),
+        apiKey: env("FOXY_WIX_API_KEY"),
+        siteId: env("FOXY_WIX_SITE_ID"),
```

Comment:
We may want to avoid prefixing 3rd party env vars with `FOXY_`. I know this is consistent with what we've done before. Just noting for the future.